content/docs/gke/pipelines-tutorial.md

```diff
@@ -1,14 +1,18 @@
 +++
 title = "Pipelines End-to-end on GCP"
 description = "An end-to-end tutorial for Kubeflow Pipelines on GCP"
-weight = 2
+weight = 15
 +++
 
 This guide walks you through a Kubeflow Pipelines sample that runs an MNIST
 machine learning (ML) model on Google Cloud Platform (GCP).
 
 ## Introductions
 
+[Kubeflow Pipelines](/docs/pipelines/) is a platform for building and 
+deploying portable, scalable machine learning (ML) workflows based on 
```

Comment:
Nit: "machine learning (ML)" already appears above, in line 8, so you could just use "ML" here

Reply:
Good point. Done.